magstock/__init__.py

```diff
@@ -45,6 +52,10 @@ class Attendee:
     def food_cost(self):
         return c.FOOD_PRICE if self.purchased_food else 0
 
+    @property
+    def auto_food(self):
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
```

Comment:
Yeah, it makes sense that you can't use a property inside a query. Annoying, though. :P